java/core/src/test/org/apache/orc/TestVectorOrcFile.java

```diff
@@ -4153,6 +4153,96 @@ public void testPredicatePushdownWithNan() throws Exception {
     assertEquals(0, batch.size);
   }
 
+  @ParameterizedTest
+  @MethodSource("data")
+  public void testPredicatePushdownWithSumOverflow(Version fileFormat) throws Exception {
```

Comment:
Ah, this is incompatible with branch-1.6. Let me fix it, @guiyanakuang .
- https://github.com/apache/orc/runs/5844990614?check_suite_focus=true

Reply:
Thanks for fixing this incompatibility. 😃